projects/storefrontlib/layout/launch-dialog/services/launch-dialog.service.ts

```diff
@@ -130,6 +130,10 @@ export class LaunchDialogService {
     return this._dialogClose.asObservable();
   }
 
+  emitData(data: any) {
```

Comment:
Could this function not be required if we call the reorder process from within the dialog?